AFQ/definitions/utils.py

```diff
@@ -0,0 +1,109 @@
+__all__ = ["Definition", "find_file"]
+
+
+class Definition():
```

Comment:
```suggestion
class Definition(object):
```

Does this even matter?

Reply:
Not in python 3, but I will do it anyways